lib/qless/version.rb

```diff
@@ -1,5 +1,5 @@
 # Encoding: utf-8
 
 module Qless
-  VERSION = '0.10.0'
+  VERSION = '0.10.1'
```

Comment:
The versioning scheme here needs to be addressed, since there are other, related repos (specifically thinking of `qless-core` and `qless-py`). I'd like to have some portion of the version space on the derived repos `qless` and `qless-py` match the version of `qless-core`. So perhaps `qless-core` gets `x.y` versions and the ruby and python bindings get `x.y.z` releases.

In the mean time, I wanted to merely bump the least-significant portion.